style.css

```diff
@@ -0,0 +1,408 @@
+  padding: 15px;
+  width: 960px;
+  margin-top: 0;
+  box-sizing: border-box;
+  z-index: 800;
+}
+.photocred {
+  font-size: 16px;
+  opacity: 0.5;
+  padding-bottom: 10px;
+}
+/*immigration*/
+.immigration {
+  width: 680px;
+  margin-left: 100px;
+  float: left;
+}
+.immigration figure {
+  margin: 0;
+  z-index: 700;
+}
+.immigration ul {
+  list-style-type: none;
+  margin: 0;
+  padding-left: 0;
+}
+.immigration li {
+  display: inline-block;
+  width: 160px;
+  padding-right: 5px;
+}
+.immigration h1 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 70px;
+  border-bottom: 10px solid black;
+  text-align: justify;
+  margin: 0;
+  text-transform: uppercase;
+}
+.immigration h3 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 26px;
+  border-bottom: 3px solid black;
+  margin: 0;
+  padding: 5px 0px;
+}
+.immigration p {
+  font-family: 'Lora', serif;
+  text-align: left;
+  font-size: 24px;
+}
+
+/*transformers*/
+.transformers {
+  max-width: 220px;
+  height: 700px;
+  margin: 20px;
+  float: left;
+  color: white;
+}
+.transformers figure {
+  margin-left: 20px;
+  position: relative;
+  bottom: 90px;
+  z-index: 900;
+}
+.transformers article {
+  padding: 180px 10px 10px 10px;
+  background-color: #3a83bb;
+  bottom: 280px;
+  z-index: 850;
+  position: relative;
+}
+.transformers header {
+  font-family: 'Oswald', sans-serif;
+  text-align: justify;
+  border-bottom: 10px solid #619bc7;
+  text-transform: uppercase;
+}
+.transformers h1 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 30px;
+  margin: 0;
+}
+.transformers h3 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 20px;
+  margin: 0 0 10px 0;
+}
+.transformers-p {
+  border-bottom: 2px dashed black;
+}
+.rating {
+  font-family: 'Oswald', sans-serif;
+  border-top: 5px solid #3476a8;
+  border-bottom: 5px solid #3476a8;
+  padding: 10px 0;
+  text-transform: uppercase;
+  width: 100%;
+}
+.rating h1 {
+  display: inline;
+}
+/*progress[value] {
+/*  background-color: #619bc7;
+ background-color: white;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.25) inset;
+}*/
+.gray {
+  color: #75a8cf;
+  margin: 0;
+}
+.date {
+  border-top: 2px dashed black;
+  font-family: 'Oswald', sans-serif;
+  text-transform: uppercase;
+  font-size: 30px;
+  padding-top: 10px;
+  width: 100%;
+}
+.date h3 {
+  margin: 0;
+}
+.premiere {
+  font-size: 10px;
+}
+
+/*overflow*/
+.overflow {
+  clear: both;
+  margin: 0;
+}
+.overflow-article {
+/*  position: relative;
+*/}
+.overflow-title {
+  font-size: 72px;
+  font-family: 'Oswald', sans-serif;
+  font-weight: 300;
+  transform: rotate(7deg);
+  color: #a0b4c0;
+  z-index: 1000;
+  margin: 0;
+  padding-left: 100px;
+  padding-top: 1px;
+  position: relative;
+  opacity: 0.6;
+}
+.water-pic {
+  z-index: 800;
+  margin-top: 0;
+  position: absolute;
+}
+.p1, .p2, .p3 {
+  font-family: 'Lora', serif;
+  color: #5f6d75;
+  width: 720px;
+
+}
+.p1 {
+  transform: rotate(2deg);
+  text-align: right;
+  position: relative;
```

Comment:
This is a common rule in all three, could have been moved up to the extended selectors.